### ISSUE TITLE
Content defined outside landmark region (#2652)

### ISSUE THREAD

**Issue by santoshsahu108** [santoshsahu108/alarm#3]
### Accessibility Issue Details

- **Issue ID**: 2652
- **Severity**: Best Practice
- **Status**: In Progress

### Page

- **Page**: santsh
- **URL**: https://barrierbreak.uknowva.com/

### Attribute

Content missing landmark

### Element Selector

```css
#wrapper
```

### Element HTML

```html
<div id="wrapper" role="main" class="mdl-layout__content  homepage ">
    <div id="wrapperouter">
	<div id="welcome_outer">
			<div class="welcometext">
			  Social			</div>

			</div>
			<div class="clr"></div>
			
		<div id="wrapperinner">
			<div id="area">
            <div id="areainner" class="mdl-grid">  
            
<div id="system-message-container">
</div>
            
      
                  
               
                                      
            
            <!--logic for dynamic width of area -->
                          
                            
              <div id="contentarea" class="mdl-cell  full mdl-cell--12-col mdl-cell--8-col-tablet mdl-cell--4-col-phone">
			  <div>
														<div class="clr"></div>
									    
				<div id="contentareainner" class="mdl-grid mdl-grid--no-spacing mdl-shadow--2dp">
				   					
					 					<div class="mdl-grid mdl-cell--12-col mdl-color--white"><div class="mainpadding">
					<div class="loginpage">
  <div class="logingreybox">
    <div class="cFrontpageSignup"> 
      <!-- Start the Login Form -->
      <div class="cFrontPageRight">
	   
        <div class="loginform">
		<h3 class="login_h3">Login</h3>
          <form action="/index.php/component/users/?task=user.login" method="post" id="form-login">
           
                                    <p class="front_label label1"><label id="username-lbl" for="username" class="">Username</label></p>
            <p class="front_input input1"><input type="text" name="username" id="username" value="" class="validate-username" size="25" placeholder="Your Username"></p>
                                                <p class="front_label label2"><label id="password-lbl" for="password" class="">Password</label></p>
            <p class="front_input input2"><input type="password" name="password" id="password" value="" class="validate-password" size="25" placeholder="Your Password"></p>
                                    <div class="buttondiv">
              <button type="submit" class="button">Login</button>
              <input type="hidden" name="return" value="aHR0cHM6Ly9iYXJyaWVyYnJlYWsudWtub3d2YS5jb20v">
            </div>
            <input type="hidden" name="73a7a7d4ef032fb258f6e69a67dd6157" value="1">          </form>
          <div>
            <ul class="loginul">
              <li class="floatl"> <a href="/index.php/reset"> Forgot your password?</a> </li>
              <li class="floatr"> <a href="/index.php/remind"> Forgot your username?</a> </li>
            </ul>
          </div>
        </div>
      </div>
      <!-- End the Login form -->
    </div>
  </div>
</div>
<script type="text/javascript">
  jQuery( document ).ready(function() {
    jQuery(".validate-username").attr("placeholder", "Your Username");
    jQuery(".validate-password").attr("placeholder", "Your Password");
  });
</script>
					</div></div>
					
				</div>
              </div></div>
                          
            </div>
          </div>
          <div class="clr"></div>
        </div>
      </div>
    <div class="clr"></div>      
    <div id="footer" class="fixed">
            <div id="footer_r"><div class="divinner">Powered by <a href="http://uknowva.com" title="uKnowva - The social collaboration software" target="_blank">uKnowva</a>
      </div>
      </div>
    </div>
    </div>
```

### WCAG Information

- **Success Criteria**: 1.3.1
- **Level**: A
- **Guidelines**: WCAG 2.2, Section 508, EN 301549

### Recommendation

<p>Use either ARIA landmark role="banner", "main", "contentinfo", "region" or "complementary" to define the content. </p>

### Actual Result Vs Expected Result

- **Actual Result**: <p>Content on the page is not defined within any landmark region, Resulting in screen reader users not being able to navigate to the content quickly.</p>
- **Expected Result**: <p>All the content should be defined within landmark regions.</p>

### Step To Validate

<p>helllo try</p>

### Suggested Code

```html
heeeeey
```

---
Created by A11yNow • Issue ID: 2652
